cmd/ebpf/pdr.go

```diff
@@ -149,12 +149,13 @@ func (bpfObjects *BpfObjects) DeleteDownlinkPdrIp6(ipv6 net.IP) error {
 }
 
 type FarInfo struct {
-	Action                uint8
-	OuterHeaderCreation   uint8
-	Teid                  uint32
-	RemoteIP              uint32
-	LocalIP               uint32
-	TransportLevelMarking uint16
+	Action                     uint8
+	OuterHeaderCreation        uint8
+	Teid                       uint32
+	RemoteIP                   uint32
+	LocalIP                    uint32
+	TransportLevelMarking      uint16
+	ForwardingPolicyIdentifier uint64
```

Comment:
Is uint64 type actually needed here, if just u32 argument in passed to `add_nsh_over_ip4_headers`
```
static __always_inline __u32 add_nsh_over_ip4_headers(struct packet_context *ctx, __u32 path_hdr)
```

Reply:
Correct, uint64 is not need, even other protocols have tags less that uint32. I have updated all the occurrences of ForwardingPolicyIdentifier and tested again.